proposals/Scoped-Custom-Element-Registries.md

```diff
@@ -60,6 +60,8 @@ registry.define('other-element', OtherElement);
 
 Definitions in this registry do not apply to the main document, and vice-versa. The registry must contain definitions for all elements used.
 
+Note that, in the above example, `<other-element>` is defined in the scoped registry, not `<my-element>`. The registry applies to definitions _within_ the shadow root, not the element that the shadow root is attached to.
```

Comment:
I'm finding this wording a bit unclear.... especially "`<other-element>` is defined in the scoped registry, not `<my-element>`" and "not the element that the shadow root is attached to."

Reply:
I tried to tweak the language a bit. Maybe I'm only muddying things further, but the existence of `<my-element>` really confused me when I first read this, so I'm trying to call that out.

Arguably this example could just attach the shadow root to a div (or something), but that might be even more confusing since it isn't a common usage of shadow DOM.